_pages/oidc.md

```diff
@@ -58,23 +58,45 @@ Consistent with the specification, login.gov provides a JSON endpoint for OpenID
 
 The authorization endpoint handles authentication and authorization of a user. To present the login.gov authorization page to a user, direct them to the `/openid_connect/authorize` endpoint with the following parameters:
 
-* **acr_values**
-  The Authentication Context Class Reference values used to specify the IAL (Identity Assurance Level) of an account, either IAL1, or IAL2. This and the `scope` determine which [user attributes]({{ site.baseurl }}/attributes/) will be available in the [user info response](#user-info-response). The possible parameter values are:
-    - `http://idmanagement.gov/ns/assurance/ial/1`
-    - `http://idmanagement.gov/ns/assurance/ial/2`
 
 
-#### Level of Assurance (LOA)
+* **acr_values**
+  The Authentication Context Class Reference requests can be used to specify the IAL (Identity Assurance Level) or the AAL (Authentication Assurance Level) for the user. These and the `scope` determine which [user attributes]({{ site.baseurl }}/attributes/) will be available in the [user info response](#user-info-response).
+
+  Multiple values can be joined with a space, before being URI-escaped. To specify strict IAL2 with AAL3:
 
-<div class="usa-alert usa-alert-warning">
-  <div class="usa-alert-body">We strongly recommend using IAL for the identity proofing process. The concept of Level of Assurance (LOA) is retired by the NIST 800-63-3 digital identity guidelines, and support by login.gov for LOA requests is deprecated.
-  </div>
-</div>
+  ```
+  http://idmanagement.gov/ns/assurance/ial/2?strict=true http://idmanagement.gov/ns/assurance/aal/3
+  ```
+  becomes this when URI-escaped:
+  ```
+  acr_values=http%3A%2F%2Fidmanagement.gov%2Fns%2Fassurance%2Fial%2F2%3Fstrict%3Dtrue+http%3A%2F%2Fidmanagement.gov%2Fns%2Fassurance%2Faal%2F2
+  ```
```

Comment:
I think I want to remove this example clause, because it's just lot of vertical noise in an already long section